src/index.js

```diff
@@ -1 +1,32 @@
-export default () => true
+import defaultsDeep from 'lodash.defaultsdeep';
+import pickBy from 'lodash.pickby';
+
+class Stipulate {
+
+  constructor(options) {
+    this.baseOptions = options;
```

Comment:
I don't see a reason to prevent people from altering the base options post-creation if they want to. If they do so instead of creating a new instance I'm inclined to assume they understand the implications.